src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const fifty = 50;
+  const twenty = 20;
```

Comment:
Bad naming.
```suggestion
  const  = 50;
  const basicDiscount = 20;
```
come up with a better name for the first variable